node/runtime/src/state_viewer.rs

```diff
@@ -21,6 +23,25 @@ pub struct StateDbViewer {
     state_db: Arc<StateDb>,
 }
 
+#[derive(Serialize, Deserialize, PartialEq, Eq, Debug)]
+pub struct AccountViewCallResult {
+    pub account: AccountId,
+    pub nonce: u64,
+    pub amount: Balance,
+    pub stake: u64,
+    pub code_hash: CryptoHash,
+}
+
+/// Result of view function call.
+#[derive(Serialize, Deserialize, PartialEq, Eq, Debug)]
+pub struct FunctionCallViewCallResult {
```

Comment:
Since it's only for the function view, we only need the result. But the result can be an error. So we may want to return an Option<> or Result<>.

Reply:
addressed